scripts/timelapse-colorizer-data/data_writer_utils.py

```diff
@@ -202,8 +228,21 @@ def write_manifest(
             "centroids": "centroids.json",
             "bounds": "bounds.json",
         }
+
+        # Merge the feature metadata together and include it in the output if present
+        if len(feature_metadata) != 0:
```

Comment:
nit: this can just be `if feature_metadata:` since empty lists are false in python